lib/rubocop/cop/style/conditional_assignment.rb

```diff
@@ -82,7 +82,7 @@ def lhs_for_send(node)
             indices = node.children[2...-1].map(&:source).join(', ')
             "#{receiver}[#{indices}] = "
           elsif node.method_name.to_s.end_with?(EQUAL) &&
-                ![:!=, :==].include?(node.method_name)
+                ![:!=, :==, :===].include?(node.method_name)
```

Comment:
Good catch. `>=` and `<=` had the same bug. I also added tests for every weird math assignment operator out there that I could find.